turbinia/evidence.py

```diff
@@ -615,3 +616,43 @@ def __init__(self, module_list=None, profile=None, *args, **kwargs):
 class BinaryExtraction(CompressedDirectory):
   """Binaries extracted from evidence."""
   pass
+
+
+class DockerContainer(Evidence):
+  """Evidence object for a DockerContainer filesystem.
+
+  Attributes:
+    container_id(str): The ID of the container to mount.
+    _container_fs_path(str): Full path to where the container filesystem will
+      be mounted.
+    _docker_root_directory(str): Full path to the docker root directory.
+  """
+
+  # ABSOLUTELY NO LEADING / HERE
+  DEFAULT_DOCKER_DIRECTORY_PATH = 'var/lib/docker'
```

Comment:
you mean being passed as a flag of turbiniactl?
I could devise a small but the directory is pretty much always that, unless configuration says otherwise. 
https://github.com/moby/moby/blob/51f27b1defad7418675567cba6aa6c95efebf739/cmd/dockerd/config_common_unix.go#L28

I guess I could also add something simple that could check if /etc/docker/daemon.json says something different

Reply:
Ok, yeah, checking the json file for the path sounds good.  We can do that in a separate PR though since it's usually the same.  Maybe file an issue for that?

Reply:
opened https://github.com/google/turbinia/issues/479